foreman.fc

```diff
@@ -51,6 +51,11 @@
 /usr/share/gems/gems/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
 /usr/lib6?4?/gems/(exts|ruby)/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
 
+# Foreman Remote Execution
+
+/usr/sbin/foreman-cockpit-session       gen_context(system_u:object_r:cockpit_session_exec_t,s0)
+/usr/share/gems/gems/foreman_remote_execution-.*/extra/cockpit/foreman-cockpit-session -- gen_context(system_u:object_r:cockpit_session_exec_t,s0)
```

Comment:
This makes me wonder if we should move the file in packaging rather than making a symlink to it in `/usr/sbin`.

Reply:
I don't mind but this is a typical pattern we use across many packages.

Reply:
Given this reflects the current package, I'm fine with merging it.